scripts/README.md

```diff
@@ -108,8 +108,10 @@ This is a helper script for Knative E2E test scripts. To use it:
    if the default values don't fit your needs:
 
    - `E2E_CLUSTER_REGION`: Cluster region, defaults to `us-central1`.
+   - `E2E_CLUSTER_BACKUP_REGION`: Backup cluster regions for retry, defaults to `us-west1 us-east1`. (Retry is only triggered when there is a stockout)
    - `E2E_CLUSTER_ZONE`: Cluster zone (e.g., `a`), defaults to none (i.e. use a regional
      cluster).
+   - `E2E_CLUSTER_BACKUP_ZONE`: Backup cluster zone for retry, defaults to none. If defined `E2E_CLUSTER_BACKUP_REGION` will be ignored.
```

Comment:
```suggestion
   - `E2E_CLUSTER_BACKUP_ZONES`: Space-separated list of zones to retry test cluster creation in case of stockout. If defined, `E2E_CLUSTER_BACKUP_REGION` will be ignored thus it defaults to none.
```

Reply:
nit: For readability and consistency, use ZONES in the var name.